packages/radix-ui-themes/changelog.md

```diff
@@ -6,6 +6,7 @@
   - Combine selectors in the CSS build, improving the developer experience when inspecting elements in the browser.
   - Remove comments from the CSS build.
   - Make sure that forced light/dark appearance on the `Theme` component also sets the corresponding browser colors, like the correct input autofill background color.
+  - Use `outline` rather than `box-shadow` for most focus styles, which avoids a slight anti-aliasing issue in Chrome on focused elements
```

Comment:
> "most"

which aren't using that?

Reply:
Slider isn't using outline. Here's our current technique with the layered box shadow:

<img width="470" alt="image" src="https://github.com/radix-ui/themes/assets/8441036/2735595e-b135-447e-aad3-1317c7a14867"> 

Here's the best that the outline can do:

<img width="470" alt="image" src="https://github.com/radix-ui/themes/assets/8441036/30f78e4d-6dc1-43b3-9b04-3589c24b3a56">